src/VisualStudio/Core/Def/Experimentation/VisualStudioExperimentationService.cs

```diff
@@ -12,48 +12,44 @@
 namespace Microsoft.VisualStudio.LanguageServices.Experimentation
 {
     [Export(typeof(VisualStudioExperimentationService))]
-    [ExportWorkspaceService(typeof(IExperimentationService), ServiceLayer.Host), Shared]
-    internal class VisualStudioExperimentationService : ForegroundThreadAffinitizedObject, IExperimentationService
+    [ExportWorkspaceService(typeof(IExperimentationServiceFactory), ServiceLayer.Host), Shared]
```

Comment:
Should this be a ExportWorkspaceServiceFactory? It's kinda funky that we're doing this.

Reply:
➡️ No, we are exporting an instance of a factory for use by consumers. We are no longer exporting instances of `IExperimentationService` or associating them directly with the workspace.

I agree the naming, though accurate, is weird.